### PR TITLE
first add jelly by owlab

### PR DESCRIPTION
**Pull request by owlab-git**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/owlab/jelly/config.h [hunk 1]
```diff
@@ -0,0 +1,106 @@
+
```
Comment (drashna) [inline keyboards/owlab/jelly/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/owlab/jelly/config.h [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+#ifndef CONFIG_H
+#define CONFIG_H
```
Comment (drashna) [inline keyboards/owlab/jelly/config.h:3]:
```suggestion
#pragma once
```

File: keyboards/owlab/jelly/config.h [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+/* USB Device descriptor parameter*/
+#define VENDOR_ID       0x4F53  //defines your VID, and for most DIY projects, can be whatever you want, 0x4F53 OS for owl studio
+#define PRODUCT_ID      0x4A53  //0x4A4C JL for jelly, 0x4A53 JS for jelly solder
+#define DEVICE_VER      0x0001 //defines the device version (often used for revisions)
+
+#define MANUFACTURER    OwLab 
+
+#define PRODUCT         JELLY
+
+#define DESCRIPTION     A custom 75% Keyboard by OwLab
```
Comment (drashna) [inline keyboards/owlab/jelly/config.h:16]:
```suggestion
#define VENDOR_ID       0x4F53  // 0x4F53 OS for owl studio
#define PRODUCT_ID      0x4A53  // 0x4A4C JL for jelly, 0x4A53 JS for jelly solder
#define DEVICE_VER      0x0001
#define MANUFACTURER    OwLab 
#define PRODUCT         JELLY
```

File: keyboards/owlab/jelly/config.h [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+
+
+
+#define DIODE_DIRECTION COL2ROW
+
+
+#define TAP_CODE_DELAY 10
+
+#define DEBOUNCING_DELAY 5
```
Comment (drashna) [inline keyboards/owlab/jelly/config.h:90]:
```suggestion
#define DEBOUNCE 5
```

File: keyboards/owlab/jelly/config.h [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+     keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
+
+
+
+
+#endif
```
Comment (drashna) [inline keyboards/owlab/jelly/config.h:106]:
```suggestion
```

File: keyboards/owlab/jelly/jelly.c [hunk 6]
```diff
@@ -0,0 +1,88 @@
+#include "jelly.h"
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/owlab/jelly/jelly.c [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+#include <print.h>
+#include "rgblight.h"
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.c:3]:
```suggestion
```

File: keyboards/owlab/jelly/jelly.c [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+
+
+void matrix_scan_kb(void) {
+    matrix_scan_user();
+}
+
+
+
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.c:88]:
```suggestion
```

File: keyboards/owlab/jelly/jelly.h [hunk 9]
```diff
@@ -0,0 +1,47 @@
+
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/owlab/jelly/jelly.h [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+#ifndef JELLY_H
+#define JELLY_H
+
+
+
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.h:6]:
```suggestion
#pragma once
```

File: keyboards/owlab/jelly/jelly.h [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+	{ K300,  K301,  K302,  K303,  K304,  K305,  K306,  K307,  K308,  K309,  K310,  K311,  K312,  K313}, \
+	{ K400,  K401,  K402,  K403,  K404,  K405,  K406,  K407,  K408,  K409,  K410,  K411,  K412,  K413}, \
+	{ K500,  K501,  K502,  K503,  K504,  K505,  K506,  K507,  K508,  K509,  K510,  K511,  K512,  K513}  \
+}
+
+
+
+
+
+#endif
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.h:47]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,79 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/owlab/jelly/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+
+}
+
+void let_set_user(uint8_t usb_led){
+	
+}
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/default/keymap.c:79]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/hotswap/keymap.c [hunk 14]
```diff
@@ -0,0 +1,79 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/hotswap/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/owlab/jelly/keymaps/hotswap/keymap.c [hunk 15]
```diff
@@ -0,0 +1,79 @@
...
+
+
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+
+}
+
+void let_set_user(uint8_t usb_led){
+	
+}
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/hotswap/keymap.c:79]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/hotswap/rules.mk [hunk 16]
```diff
@@ -0,0 +1,3 @@
+
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/hotswap/rules.mk:3]: VIA should only be enabled for the `via` keymap. 
```suggestion
```

File: keyboards/owlab/jelly/keymaps/solder/keymap.c [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+
+
+
+void encoder_update_user(uint8_t index, bool clockwise) {
+
+}
+
+void let_set_user(uint8_t usb_led){
+	
+}
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/solder/keymap.c:78]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/solder/rules.mk [hunk 18]
```diff
@@ -0,0 +1,3 @@
+
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/solder/rules.mk:3]:
```suggestion
```

File: keyboards/owlab/jelly/rules.mk [hunk 19]
```diff
@@ -0,0 +1,68 @@
...
+SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+
+
+
+
+
+
```
Comment (drashna) [inline keyboards/owlab/jelly/rules.mk:68]:
```suggestion
# MCU name 
MCU = STM32F303

# Bootloader selection
BOOTLOADER = stm32-dfu

# Build Options
#   comment out to disable the options.
#
BOOTMAGIC_ENABLE = lite  # Virtual DIP switch configuration(+1000) lizm
MOUSEKEY_ENABLE = yes    # Mouse keys(+4700) lizm
EXTRAKEY_ENABLE = yes    # Audio control and System control(+450)


CONSOLE_ENABLE = no      # Console for debug(+400) lizm
COMMAND_ENABLE = no      # Commands for debug and configuration lizm
SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
NKRO_ENABLE = yes        # USB Nkey Rollover
BACKLIGHT_ENABLE = no    # Enable keyboard backlight functionality
AUDIO_ENABLE = no

RGBLIGHT_ENABLE = yes 
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/owlab/jelly/config.h (hunk 4 above)
Comment (drashna) [reply keyboards/owlab/jelly/config.h:82]:
```suggestion
```

File: keyboards/owlab/jelly/config.h [hunk 20]
```diff
@@ -0,0 +1,95 @@
...
+// #define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+// #define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+     keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
+#endif
```
Comment (drashna) [inline keyboards/owlab/jelly/config.h:95]:
```suggestion
```

File: keyboards/owlab/jelly/jelly.c [hunk 21]
```diff
@@ -0,0 +1,74 @@
...
+        default:
+        break;
+    }
+  }
+  return process_record_user(keycode, record);
+}
+void matrix_init_kb(void) {
+    matrix_init_user(); 
+    
+}
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.c:59]:
```suggestion
```

File: keyboards/owlab/jelly/jelly.c [hunk 22]
```diff
@@ -0,0 +1,74 @@
...
+        rgblight_mode(RGBLIGHT_MODE_STATIC_LIGHT);
+    }
+}
+void matrix_scan_kb(void) {
+    matrix_scan_user();
+}
+
+
+
+
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.c:73]:
```suggestion
```

File: keyboards/owlab/jelly/jelly.h [hunk 23]
```diff
@@ -0,0 +1,61 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef JELLY_H
+#define JELLY_H
```
Comment (drashna) [inline keyboards/owlab/jelly/jelly.h:17]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/default/keymap.c [hunk 24]
```diff
@@ -0,0 +1,70 @@
...
+	return true;
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void let_set_user(uint8_t usb_led){
+	
+}
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/default/keymap.c:70]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/solder/keymap.c [hunk 25]
```diff
@@ -0,0 +1,56 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/solder/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/owlab/jelly/keymaps/solder/keymap.c [hunk 26]
```diff
@@ -0,0 +1,56 @@
...
+	return true;
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void let_set_user(uint8_t usb_led){
+	
+}
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/solder/keymap.c:56]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/via/keymap.c [hunk 27]
```diff
@@ -0,0 +1,70 @@
...
+	return true;
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void let_set_user(uint8_t usb_led){
+	
+}
```
Comment (drashna) [inline keyboards/owlab/jelly/keymaps/via/keymap.c:70]:
```suggestion
```

File: keyboards/owlab/jelly/rules.mk [hunk 28]
```diff
@@ -0,0 +1,23 @@
...
+NKRO_ENABLE = yes        # USB Nkey Rollover
+BACKLIGHT_ENABLE = no    # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes 
+
+
+
+
+
+
```
Comment (drashna) [inline keyboards/owlab/jelly/rules.mk:23]:
```suggestion
````

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/owlab/jelly/config.h (hunk 1 above)
Comment (owlab-git) [reply keyboards/owlab/jelly/config.h:1]: Already Edited.

**Comment by owlab-git** [comment]
bump

**Comment by zvecr** [comment]
You still have unresolved comments, bumping in this case is still on you.

File: keyboards/owlab/jelly/readme.md [hunk 29]
```diff
@@ -0,0 +1,16 @@
...
+A 75% made by Owlab.
+
+Keyboard Maintainer: [captsis](https://github.com/captsis), [owlab](https://github.com/owlab-git)   
+Hardware Supported: Jelly Epoch powered by the STM32F303  
+Hardware Availability: [zFrontier](https://www.zfrontier.com/app/flow/DAVlNnzvEKY8), [Geekhack GB](https://geekhack.org/index.php?topic=111841.0)
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make jelly:default
```
Comment (zvecr) [inline keyboards/owlab/jelly/readme.md:14]:
```suggestion
    make owlab/jelly:default
```

File: keyboards/owlab/jelly/keymaps/via/rules.mk [hunk 30]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+BOOTMAGIC_ENABLE = lite
```
Comment (zvecr) [inline keyboards/owlab/jelly/keymaps/via/rules.mk:2]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/via/keymap.c [hunk 31]
```diff
@@ -0,0 +1,61 @@
...
+};
+void matrix_init_user(void) {
+}
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	return true;
+}
+void matrix_scan_user(void) {
+}
+void let_set_user(uint8_t usb_led){
+}
```
Comment (zvecr) [inline keyboards/owlab/jelly/keymaps/via/keymap.c:61]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/solder/keymap.c [hunk 32]
```diff
@@ -0,0 +1,61 @@
...
+};
+void matrix_init_user(void) {
+}
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	return true;
+}
+void matrix_scan_user(void) {
+}
+void let_set_user(uint8_t usb_led){	
+}
```
Comment (zvecr) [inline keyboards/owlab/jelly/keymaps/solder/keymap.c:61]:
```suggestion
```

File: keyboards/owlab/jelly/keymaps/default/keymap.c [hunk 33]
```diff
@@ -0,0 +1,61 @@
...
+};
+void matrix_init_user(void) {
+}
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	return true;
+}
+void matrix_scan_user(void) {
+}
+void let_set_user(uint8_t usb_led){
+}
```
Comment (zvecr) [inline keyboards/owlab/jelly/keymaps/default/keymap.c:61]:
```suggestion
```

File: keyboards/owlab/jelly/jelly.c [hunk 34]
```diff
@@ -0,0 +1,66 @@
...
+// set the first rgb animation to RGBLIGHT_MODE_STATIC_LIGHT
+void keyboard_post_init_kb(void){
+   
+    if(rgblight_get_mode()==RGBLIGHT_MODE_STATIC_LIGHT){
+        rgblight_mode(RGBLIGHT_MODE_STATIC_LIGHT);
+    }
+}
+void matrix_scan_kb(void) {
+    matrix_scan_user();
+}
```
Comment (zvecr) [inline keyboards/owlab/jelly/jelly.c:66]:
```suggestion
```

File: keyboards/owlab/jelly/info.json [hunk 35]
```diff
@@ -0,0 +1,95 @@
+{
+    "keyboard_name": "jelly",
+    "url": "https://www.popkeyboard.com/products/jrrgvdsyu47wer",
+    "maintainer": "Owlab",
+    "width": 16,
+    "height": 6.25,
+    "layouts": {
+        "LAYOUT": {
```
Comment (zvecr) [inline keyboards/owlab/jelly/info.json:8]: This does not match the content of jelly.h

File: keyboards/owlab/jelly/config.h [hunk 36]
```diff
@@ -0,0 +1,95 @@
...
+
+
+
+
+#define DIODE_DIRECTION COL2ROW
+
+
+#define DEBOUNCING_DELAY 5
+#define DEBOUNCE 5
```
Comment (zvecr) [inline keyboards/owlab/jelly/config.h:83]:
```suggestion
#define DEBOUNCE 5
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/owlab/jelly/config.h (hunk 1 above)
Comment (zvecr) [reply keyboards/owlab/jelly/config.h:1]: > Already Edited.

Nope, still needs doing on this file

File: keyboards/owlab/jelly/info.json (hunk 35 above)
Comment (owlab-git) [reply keyboards/owlab/jelly/info.json:8]: This is actually match the content of Jelly.h. In order to reduce pins in wiring, Delete, Home and other keys are arranged on other rows. Attached here is the physical layout picture and the circuit schematic diagram for your reference.
[https://cdn.shopifycdn.net/s/files/1/0508/8716/4061/files/e8661ef182b17d75892f6664ea3581d9.jpg?v=1618594274](url)
[https://cdn.shopifycdn.net/s/files/1/0508/8716/4061/files/Jelly_hotswap_circuit_schematic.png?v=1629449798](url)

Comment (owlab-git) [reply keyboards/owlab/jelly/info.json:8]: https://cdn.shopifycdn.net/s/files/1/0508/8716/4061/files/e8661ef182b17d75892f6664ea3581d9.jpg?v=1618594274
https://cdn.shopifycdn.net/s/files/1/0508/8716/4061/files/Jelly_hotswap_circuit_schematic.png?v=1629449798

**Comment by owlab-git** [comment]
Modified.

File: keyboards/owlab/jelly/config.h (hunk 1 above)
Comment (drashna) [reply keyboards/owlab/jelly/config.h:1]: if you're editing locally, double check it and make sure that you've pushed any changes.

File: keyboards/owlab/jelly/config.h [hunk 37]
```diff
@@ -0,0 +1,94 @@
...
+
+
+
+#define RGB_DI_PIN B15
+
+// #define WS2812_SPI SPID2
+// #define WS2812_SPI_MOSI_PAL_MODE 5
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/owlab/jelly/config.h:65]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/owlab/jelly/rules.mk [hunk 38]
```diff
@@ -0,0 +1,17 @@
...
+BOOTMAGIC_ENABLE = lite  # Virtual DIP switch configuration(+1000) lizm
+MOUSEKEY_ENABLE = yes    # Mouse keys(+4700) lizm
+EXTRAKEY_ENABLE = yes    # Audio control and System control(+450)
+CONSOLE_ENABLE = no      # Console for debug(+400) lizm
+COMMAND_ENABLE = no      # Commands for debug and configuration lizm
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes        # USB Nkey Rollover
+BACKLIGHT_ENABLE = no    # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
```
Comment (drashna) [inline keyboards/owlab/jelly/rules.mk:17]:
```suggestion
# MCU name 
MCU = STM32F303

# Bootloader selection
BOOTLOADER = stm32-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite  # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes    # Mouse keys
EXTRAKEY_ENABLE = yes    # Audio control and System control
CONSOLE_ENABLE = no      # Console for debug
COMMAND_ENABLE = no      # Commands for debug and configuration
SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
NKRO_ENABLE = yes        # USB Nkey Rollover
BACKLIGHT_ENABLE = no    # Enable keyboard backlight functionality
AUDIO_ENABLE = no        # Enables audio
RGBLIGHT_ENABLE = yes    # Enables RGB underglow
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/owlab/jelly/config.h (hunk 37 above)
Comment (owlab-git) [reply keyboards/owlab/jelly/config.h:65]: Hi, all the animations are enabled. Should I define all animations?

Comment (owlab-git) [reply keyboards/owlab/jelly/config.h:65]: Hi, all the animations are enabled. Should I define all animations?

**Comment by kb-elmo** [comment]
I am currently working on a rewrite of the files for this board (in agreement with OwLab).
I would suggest that this PR gets closed in the meantime.

The files are pretty much completely wrong and don't match the hardware of the keyboard at all (it doesn't have any RGB matrix yet that stuff is defined in the board files for example).

File: keyboards/owlab/jelly/info.json (hunk 35 above)
Comment (zvecr) [reply keyboards/owlab/jelly/info.json:8]: https://www.reddit.com/r/confidentlyincorrect -  LAYOUT doesnt exist for this keyboard

**Pull request closed without merging**